https://github.com/martijnvanbrummelen/nwipe/pull/516
Move the definition of the off64_t using int64_t bellow the include stdint.h, which is actually defining it.
Without change wrong order is breaking the build on RHEL7/8/9.

diff -ru nwipe-0.35/src/nwipe.h nwipe-0.35.new/src/nwipe.h
--- nwipe-0.35/src/nwipe.h	2023-11-02 20:19:21.000000000 +0000
+++ nwipe-0.35.new/src/nwipe.h	2023-11-03 21:33:11.451155596 +0000
@@ -36,15 +36,6 @@
 #define _FILE_OFFSET_BITS 64
 #endif
 
-/* workaround for Fedora */
-#ifndef off64_t
-#ifndef off_t
-#define off64_t int64_t
-#else
-#define off64_t off_t
-#endif
-#endif
-
 /* Busybox headers. */
 #ifdef BB_VER
 #include "busybox.h"
@@ -68,6 +59,15 @@
 #include <time.h>
 #include <unistd.h>
 
+/* workaround for Fedora */
+#ifndef off64_t
+#ifndef off_t
+#define off64_t int64_t
+#else
+#define off64_t off_t
+#endif
+#endif
+
 /*#include "config.h"*/
 
 /* System errors. */
